global-registry-bindings.gemspec

```diff
@@ -23,20 +23,20 @@ Gem::Specification.new do |s|
 
   s.add_runtime_dependency 'activerecord', '>= 4.0.0', '< 7'
   s.add_runtime_dependency 'global_registry', '~> 1.4', '< 2'
-  s.add_runtime_dependency 'sidekiq', '>= 4.0.0', '< 6'
-  s.add_runtime_dependency 'sidekiq-unique-jobs', '>= 5.0.0', '< 6'
+  s.add_runtime_dependency 'sidekiq', '>= 4.0.0', '< 7'
```

Comment:
Should we change this to `>= 5.0.0` because we're not testing in that? apps using sidekiq 4.x should use an old version of global-registry-bindings

Reply:
Yup, good call.